### PR TITLE
fix(CX-3661): Correct Urgency Info text color

### PR DESCRIPTION
**Pull request by lordkiz**
This PR resolves [CX-3661] 

### Description
- Correct Urgency Info text color
<p>

</p>

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Correct Urgency Info text color - kizito

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3661]: https://artsyproduct.atlassian.net/browse/CX-3661?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MounirDhahri — DISMISSED** [review]

**Comment by MounirDhahri** [comment]
🚀

File: src/app/Components/SaleArtworkTileRailCard/UrgencyInfo.tests.tsx [hunk 1]
```diff
@@ -45,4 +45,27 @@ describe("UrgencyInfo", () => {
...
+      const start = new Date(new Date().getTime() - 10).toISOString()
+      const end = new Date(new Date().getTime() + 2 * HR).toISOString()
+      const { getByText } = renderWithWrappers(
+        <UrgencyInfo startAt={start} endAt={end} saleTimeZone="America/New_York" />
+      )
+
+      jest.advanceTimersByTime(2000)
+      expect(getByText("1h 59m left")).toHaveStyle({ color: "#1023D7" })
+    })
+  })
```
Comment (MrSltun) [inline src/app/Components/SaleArtworkTileRailCard/UrgencyInfo.tests.tsx:70]: 👏👏

**Review by MrSltun — APPROVED** [review]
Looks great! Thanks Kizito!! 🌟 
I just left one suggestion but it's not blocking :)

File: src/app/Components/SaleArtworkTileRailCard/UrgencyInfo.tsx [hunk 2]
```diff
@@ -43,7 +42,7 @@ export const UrgencyInfo: React.FC<UrgencyInfoProps> = (props) => {
   return null
 }
 
-const getInfo = (time: Time) => {
+const getInfo = (time: Time): { copy: string; color: "blue100" | "red100" } => {
```
[the remark this answers is not in the document]
Comment (lordkiz) [reply src/app/Components/SaleArtworkTileRailCard/UrgencyInfo.tsx:45]: @MrSltun Yessss. I will do that.

**Pull request merged**